K2Bridge.Tests.UnitTests/Visitors/AggregationVisitorTests.cs

```diff
@@ -86,5 +86,37 @@ public string AggregationVisit_WithSubCardinalityAggs_ReturnsDCounts()
 
             return aggregateClause.KustoQL;
         }
+
+        [TestCase(ExpectedResult = "_A=sum(fieldA)")]
```

Comment:
See comment about quoting field aliases...

```suggestion
        [TestCase(ExpectedResult = "['_A']=sum(fieldA)")]
```

Reply:
Fixed.